pl_examples/basic_examples/mnist_datamodule.py

```diff
@@ -55,6 +55,9 @@ def __init__(
             normalize: If true applies image normalize
         """
         super().__init__(*args, **kwargs)
+        if platform.system() == "Windows":
+            # see: https://stackoverflow.com/a/59680818/4521646
```

Comment:
```suggestion
            # see: https://stackoverflow.com/a/59680818
```

also, can you set this 5 lines below where it is also set?

```
# see: https://stackoverflow.com/a/59680818
self.num_workers = num_workers if platform.system() != "Windows" else 0
```

Reply:
Not sure, but I thought this would be supported now since Microsoft take over PT support for windows?

Reply:
let's keep this one and warning